src/styles/main.scss

```diff
@@ -1,3 +1,58 @@
+$calendar-item-length: 100px;
+$calendar-length: ($calendar-item-length * 7 + 20);
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  max-width: #{$calendar-length};
+  padding: 10px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  gap: 1px;
+  flex-wrap: wrap;
+  &__item {
+    box-sizing: border-box;
+    width: $calendar-item-length;
+    height: $calendar-item-length;
+    border: 1px solid black;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: #eee;
+    transition: all 0.5s ease;
```

Comment:
it's better (for performance) to list only properties you really need instead of `all` in transition